### PR TITLE
Add ar to translations.ts

### PR DESCRIPTION
**Pull request by X7md**
Not sure about RTL implementation, but arrow might have reverse arrow I think.

+ Will be nice if someone review the translation before marge it...

### CODE REVIEW

**Comment by trueberryless** [comment]
Thank you very much for your contribution!

I cannot answer the question with the arrow, but does this looks correct to you or does the arrow need to point to the left?

image

We can just edit the arrow if it is necessary...

**Comment by X7md** [comment]
> Thank you very much for your contribution!
> 
> I cannot answer the question with the arrow, but does this looks correct to you or does the arrow need to point to the left?
> 
> image
> 
> We can just edit the arrow if it is necessary...

From the picture arrows should be reversed... Because that how RTL language should looks like

**Comment by trueberryless** [comment]
Perfect, now it looks like this:

image

File: packages/starlight-cooler-credit/translations.ts [hunk 1]
```diff
@@ -55,4 +55,11 @@ export const Translations = {
         "starlightCoolerCredit.starlight-blog.title": "由 Starlight Blog 构建",
         "starlightCoolerCredit.starlight-blog.description": "想建立自己的 Starlight 博客吗？ →",
     },
+    ar: {
+        "starlightCoolerCredit.starlight.description": "تود بناء توثيقك الخاص؟ ←",
+        "starlightCoolerCredit.astro.title": "مبني على Astro",
+        "starlightCoolerCredit.astro.description": "تود بناء موقعك الثابت؟ ←",
+        "starlightCoolerCredit.starlight-blog.title": "مبني على Starlight Blog",
+        "starlightCoolerCredit.starlight-blog.description": "تود بناء مدونتك الخاصة على Starlight Blog؟ ←",
```
Comment (trueberryless) [inline packages/starlight-cooler-credit/translations.ts:63]: Actually the Starlight Blog here is not a proper name, so blog should be translated into Arabic as well... I have no idea if the suggestion is correct though

**Review by trueberryless — COMMENTED** [review]
I reviewed it as a non-native speaker with a translator app and it generally looks good to me, but the last translation is a little bit misleading I think.

File: packages/starlight-cooler-credit/translations.ts (hunk 1 above)
Comment (trueberryless) [inline packages/starlight-cooler-credit/translations.ts:63]:
```suggestion
        "starlightCoolerCredit.starlight-blog.description": "هل تريد ان تطور Starlight blog الخاص بك؟ ←",
```

**Review by trueberryless — APPROVED** [review]
LGMT! 🎉 

@X7md and @OmarIsAdev, thank you guys very much for the Arabic translation!

**Pull request merged**